administrator/language/de-DE/plg_system_webauthn.ini

```
@@ -0,0 +1,47 @@
+
+PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Web-Authentifizierung"
+PLG_SYSTEM_WEBAUTHN_LOGIN_DESC="Ohne Passwort mit dem W3C Web-Authentifizierungsstandard (WebAuthn) in kompatiblen Browsern anmelden. Die WebAuthn-Authentifizierung muss vorab im Benutzerprofil eingerichtet werden."
+
+PLG_SYSTEM_WEBAUTHN_HEADER="W3C Web-Authentifizierung (WebAuthn) Anmeldung"
+PLG_SYSTEM_WEBAUTHN_FIELD_LABEL="W3C Web-Authentifizierung (WebAuthn) Anmeldung"
+PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Ermöglicht die Verwaltung kennwortloser Anmeldeverfahren unter Verwendung des W3C-Web-Authentifizierungsstandards. Benötigt wird ein unterstützten Browser und Authentifikator (z.B. Google Chrome oder Firefox mit einem FIDO2-zertifizierten Sicherheitsschlüssel)."
+
+PLG_SYSTEM_WEBAUTHN_MANAGE_FIELD_KEYLABEL_LABEL="Name des Authentifizierers"
+PLG_SYSTEM_WEBAUTHN_MANAGE_FIELD_KEYLABEL_DESC="Ein kurzer Name für den Authentifikator, der bei dieser kennwortlosen Anmeldemethode verwendet wird."
+PLG_SYSTEM_WEBAUTHN_MANAGE_HEADER_NOMETHODS_LABEL="Es wurden noch keine Authentifizierer eingerichtet."
+PLG_SYSTEM_WEBAUTHN_MANAGE_HEADER_ACTIONS_LABEL="Aktionen"
+PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_DELETE_LABEL="Entfernen"
+PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_EDIT_LABEL="Name bearbeiten"
+PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_ADD_LABEL="Neuen Authenticator hinzufügen"
+PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_SAVE_LABEL="Speichern"
+PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_CANCEL_LABEL="Abbrechen"
+
+PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR_LABEL="Authentifikator wurde am %s hinzugefügt"
+
+PLG_SYSTEM_WEBAUTHN_MSG_SAVED_LABEL="Der Name wurde gespeichert."
+PLG_SYSTEM_WEBAUTHN_MSG_DELETED="Der Authentifikator wurde entfernt."
+
+PLG_SYSTEM_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Die gespeicherten Anmeldedaten für diesen Login-Authentifikator können nicht gefunden werden."
+PLG_SYSTEM_WEBAUTHN_ERR_CORRUPT_STORED_CREDENTIAL="Die gespeicherten Anmeldeinformationen für dieses Benutzerkonto sind beschädigt. Bitte mit einer anderen Methode anmelden, und den Login-Authentifikator entfernen und erneut hinzufügen."
+PLG_SYSTEM_WEBAUTHN_ERR_CANT_STORE_FOR_GUEST="Kann möglicherweise keine Anmeldeinformationen für Gastbenutzer speichern!"
+PLG_SYSTEM_WEBAUTHN_ERR_CREDENTIAL_ID_ALREADY_IN_USE="Kann keine Anmeldedaten speichern. Diese Zugangsdaten werden bereits von einem anderen Benutzer verwendet."
+PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="Der Benutzer für diesen Authentifikator scheint auf dieser Site nicht mehr zu existieren."
+PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Leider unterstützt der aktuelle Browser nicht den W3C-Web-Authentifizierungsstandard für kennwortlose Anmeldungen oder diese Website wird nicht über HTTPS mit einem gültigen Zertifikat, das von einer Zertifizierungsstelle dieses Browsers signiert ist, bereitgestellt. Bitte mit Benutzernamen und Passwort auf dieser Website anmelden."
```

> das von einer Zertifizierungsstelle dieses Browsers signiert ist ?

Does a browser have a certification authority? I think the sentence is not clearly understandable for an end user.

---

so würde ich es sehen: https://security.stackexchange.com/questions/49006/list-of-certificate-authorities-in-browsers-and-mobile-platforms

ggf 
```suggestion
PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Leider unterstützt der aktuelle Browser nicht den W3C-Web-Authentifizierungsstandard für kennwortlose Anmeldungen oder diese Website wird nicht über HTTPS mit einem gültigen Zertifikat, das von einer Zertifizierungsstelle welche der Browser vertraut signiert ist, bereitgestellt. Bitte mit Benutzernamen und Passwort auf dieser Website anmelden."
```

---

```suggestion
PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Leider unterstützt der aktuelle Browser nicht den W3C-Web-Authentifizierungsstandard für passwortlose Anmeldungen oder diese Website wird nicht über HTTPS mit einem gültigen Zertifikat, das von einer Zertifizierungsstelle, welche der Browser vertraut, signiert ist, bereitgestellt. Bitte mit Benutzernamen und Passwort auf dieser Website anmelden."
```

---

Aber das Satz ist Horror! Der kann so nicht bleiben.